v2/openapi/fb-unified-openapi.yaml

```diff
@@ -2905,13 +2933,18 @@ components:
               type: string
             amount:
               $ref: '#/components/schemas/PositiveAmount'
```

Comment:
Split it into `InternalTransferAddress` that contains only the account ID and `InternalTransferDestination` that inherits `InternalTransferAddress` and adds to it the amount property.

Reply:
Done